src/PKSim.Presentation/Presenters/Simulations/SimulationParametersPresenter.cs

```diff
@@ -53,5 +59,26 @@ private bool canHandle(SimulationUpdatedEvent eventToHandle)
       {
          return Equals(_simulation, eventToHandle.Simulation);
       }
+
+      public void Handle(SimulationRunStartedEvent eventToHandle)
+      {
+         if (!_simulation.Equals(eventToHandle.Simulation))
+            return;
+
+         _view.SetParametersTabEnabled(false);
+      }
+
+      public void Handle(SimulationRunFinishedEvent eventToHandle)
+      {
+         if (!_simulation.Equals(eventToHandle.Simulation))
+            return;
+
+         _view.SetParametersTabEnabled(true);
+      }
+
+      public void Handle(SimulationRunCanceledEvent eventToHandle)
+      {
```

Comment:
why don't you check for the simulation in this case to verify that this is the one being canceled?